Fixes a sysbuild failure for M5Stack CoreS3

soburi opened the pull request:
This is caused by a driver whose dependencies cannot be resolved being included during sysbuild.

mfd_aw9523b, gpi_aw9523b: Replace 'select I2C' with 'depends on I2C' in Kconfig to follow the specified build settings.

ft5336: Enables INPUT under stricter conditions (when LVGL or DISPLAY is enabled).

sylvioalves commented on boards/m5stack/m5stack_cores3/Kconfig.defconfig:
```diff
@@ -9,6 +9,6 @@ config INPUT_FT5336_INTERRUPT
 
 config INPUT
-	default y
+	default y if LVGL || DISPLAY
```
Should this be also addressed for `m5stack_core2` board?

soburi replied:
On CoreS3/CoreS3SE, the I2C-connected GPIO Expander AW9523B is connected to various parts of the system, so it should be enabled at all times, but this enables I2C, which causes a conflict with the mcuboot settings.
On Core2, there is no GPIO Expander, so no problems occur.

sylvioalves commented on boards/m5stack/m5stack_cores3/m5stack_cores3_procpu_defconfig:
```diff
@@ -5,3 +5,5 @@ CONFIG_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_UART_CONSOLE=y
 CONFIG_GPIO=y
+CONFIG_I2C=y
```
I think `CONFIG_I2C=y` isn't necessary as it is automatically added when enabled in DTS file. Can you confirm whether you need this here?

soburi replied:
Thank you for your comment.

I found the correct way to fix it.
The device can be disabled in the MCUBoot settings, so I changed it to deal with the problem that way.

soburi committed:
boards: m5stack: cores3: Label aw9523b for easy reference from overlays

Add this to disable on mcuboot.

Signed-off-by: TOKITA Hiroshi <tokita.hiroshi@gmail.com>

kartben commented:
@soburi please update manifest with final sha

soburi commented:
@kartben 

> @soburi please update manifest with final sha

https://github.com/zephyrproject-rtos/mcuboot/pull/122

I will remove DNM after this PR is merged.

nordicjm committed:
west.yml: MCUboot synchronization from upstream

Update Zephyr fork of MCUboot to revision:
  06747985eee7c27d891f2529c58e18f4a84ea6a3

Brings following Zephyr relevant fixes:
  - 06747985 zephyr: Remove scratch from
    flash_area_id_from_multi_image_slot
  - b18b91c2 boot: boot_serial: Fix uninitialised variables for
    upload
  - 1da18e90 scripts: imgtool: fix compression with encryption
  - 256a02c3 added missing MCUBOOT_VERSION_TWEAK in Zephyr port
  - 4bda5875 boot: bootutil: Add TLV for size of compressed but
    decrypted image
  - 1c04eac6 imgtool: Add pure signature support
  - 95773edf boot: zephyr: boards: add support for M5stack
    CoreS3/CoreS3SE

Signed-off-by: Jamie McCrae <jamie.mccrae@nordicsemi.no>

nordicjm commented on west.yml:
```diff
@@ -303,7 +303,7 @@ manifest:
         - crypto
     - name: mcuboot
-      revision: a2bc982b3379d51fefda3e17a6a067342dce1a8b
+      revision: 95773edf33082a1689795e0278c14b2da1e6ab74
```
replace commit with commit from https://github.com/nordicjm/zephyr/tree/mcubootcherrypick

soburi replied:
Thank you for your suggestion.
I have extracted your patch, applied it, and pushed the changes.
Could you take a look?

soburi commented:
@sylvioalves @henrikbrixandersen 
Can you re-approve this?

soburi commented:
@sylvioalves @henrikbrixandersen 
Could you re-approve if there is no problem?

sylvioalves commented:
You also need @soburi to approve.

kartben commented:
> You also need @soburi to approve.

@sylvioalves he's the author 😬

sylvioalves commented:
> > You also need @soburi to approve.
> 
> @sylvioalves he's the author 😬

hahahahaha, of course. I see he is also the assignee, that's why I mentioned it. My bad.

soburi commented:
Thanks, all!
